frontend/src/common/queries/cellGuide.ts

```diff
@@ -395,6 +395,52 @@ export const fetchTissueMetadata =
     return await response.json();
   };
 
+/* ========== Lookup tables for organs ========== */
+export function useAllOrgansLookupTables(): {
```

Comment:
Since this is just returning one thing, we don't need to return it as an object. Just return the thing directly.

Reply:
will fix